docs/contributor_guide.rst

```diff
@@ -99,7 +99,8 @@ Existing tags:
   Therefore the changelog will distinguish `what the user should know` from `what the developers are doing`.
   This is done by **capitalizing PR tags**:
 
-   - PRs with visible/notable features to the users should be marked with tags starting with **the first letter capitalized**, e.g. ``[Metal], [OpenGL], [IR], [Lang], [CLI]``;
+   - PRs with visible/notable features to the users should be marked with tags starting with **the first letter capitalized**, e.g. ``[Metal], [OpenGL], [IR], [Lang], [CLI]``.
```

Comment:
Thanks! Could you also place these in PR template?

Reply:
Good point. Added a brief version and a link.